src/features/tasks/components/TaskActionButtons/index.tsx

```diff
@@ -57,34 +70,25 @@ const TaskActionButtons: React.FunctionComponent<TaskActionButtonsProps> = ({
         <ZUIEllipsisMenu
           items={[
             {
-              id: TASK_MENU_ITEMS.EDIT_TASK,
-              label: (
-                <>
-                  <Box mr={1}>
-                    <Settings />
-                  </Box>
-                  <Msg id={messageIds.editTask.title} />
-                </>
-              ),
+              label: messages.actions.move(),
+              onSelect: () => setIsMoveDialogOpen(true),
+              startIcon: <ArrowForward />,
+            },
+            {
+              label: messages.editTask.title(),
               onSelect: () => setEditTaskDialogOpen(true),
+              startIcon: <Settings />,
             },
             {
-              id: TASK_MENU_ITEMS.DELETE_TASK,
```

Comment:
The `id` properties on these menu items might look like they did not do anything special, but they are used for testing purposes. Removing them is making the integration tests fail :)